.github/workflows/build.yml

```diff
@@ -0,0 +1,78 @@
+name: Build
+on:
+  push:
+    branches:
+    - master
+    tags:
+    - "*"
+  pull_request:
+env:
+  Z3_COMMIT: d5c6abe14d3edc18667e5828144816ec756289fb
+  TARGET_FRAMEWORK: netstandard1.4
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+        - os: ubuntu-18.04
+          makeScript: python scripts/mk_unix_dist.py --nojava --dotnet-key=resources/z3.snk
+        - os: windows-2019
+          makeScript: cmd /c "`"C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvarsall.bat`" x64 && python scripts\mk_win_dist.py --x64-only --nojava --dotnet-key=resources/z3.snk"
```

Comment:
I tried changing `` `"C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvarsall.bat`" `` to both `` `"${(Get-VsSetupInstance).InstallationPath}\VC\Auxiliary\Build\vcvarsall.bat`" `` and `` `"${vswhere -property installationPath}\VC\Auxiliary\Build\vcvarsall.bat`" `` to avoid having to hardcode the VS installation location here. Either or both might work, but I don't think the interpolation is even being expanded. I have too many layers of PowerShell and cmd going on to know what is happening, so I just put it back to the simplest way. This matches what [Z3](https://github.com/Z3Prover/z3/blob/d5c6abe14d3edc18667e5828144816ec756289fb/scripts/build-win-signed.yml#L20) do themselves anyway.

Reply:
I think this is good enough tbh. If it breaks it should be very obvious why and these paths, whilst long fiddly are pretty stable afaik.